### PR TITLE
feat: Add employee saved filters `POST` API docs

### PR DESCRIPTION
**Pull request by RoshanMhatre**
### Adding docs for `/admin/employees/filters` POST API docs

https://app.clickup.com/t/85ztbhzqq

### CODE REVIEW

File: src/components/schemas/fields.yaml [hunk 1]
```diff
@@ -1302,3 +1302,27 @@ mileage_settings:
...
+  default: false
+  example: true
+
+query_string:
+  type: string
+  nullable: false
+  maxLength: 255
+  description: |
+    This represents a query string which will be used to filter employees.
+  example: or=(and(code.eq.'E0101'))
```
Comment (jayachandhar) [inline src/components/schemas/fields.yaml:1328]: `or` part is not used, either add expression for `and` & `or`, else we can keep it simple, eg `code.eq.'E0101'`

File: src/components/schemas/employee_filters.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+employee_filters_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: filtumWHrlPf7K
```
Comment (jayachandhar) [inline src/components/schemas/employee_filters.yaml:23]: efil

**Review by jayachandhar — APPROVED** [review]

File: src/components/schemas/fields.yaml (hunk 1 above)
Comment (RoshanMhatre) [reply src/components/schemas/fields.yaml]: Got it, updated the docs
Thanks!

File: src/components/schemas/employee_filters.yaml (hunk 2 above)
Comment (RoshanMhatre) [reply src/components/schemas/employee_filters.yaml]: done

**Pull request merged**